content/design-systems-for-developers/react/en/test.md

```diff
@@ -60,64 +62,29 @@ npx chromatic --project-token=<project-token>
 ```
 
 ![Result of our first Chromatic build](/design-systems-for-developers/chromatic-first-build.png)
+-->
 
-Chromatic captured a baseline image of every story! Subsequent test runs will capture new images and compare them against these baselines. See how that works by tweaking a UI component and saving it. Go to the global styles (`src/shared/styles.js`) and increase the font-size.
+When the pull request was created, Chromatic published Storybook then captured a set of images for the changes that we made!
 
-```javascript
-// …
-export const typography = {
-  // ...
-  size: {
-    s1: '13',
-    // ...
-  },
-};
-// ...
-```
+![List of checks in the pull request](/design-systems-for-developers/chromatic-list-of-checks.png)
 
-Run the test command again.
+Click the "🟡 UI Tests" check to review them.
 
-```bash
-npx chromatic --project-token=<project-token>
-```
+![Second build in Chromatic with changes](/design-systems-for-developers/chromatic-second-build-from-pr.png)
```

Comment:
iirc the button component has 3 stories. Why are only 2 of them registering changes?

Reply:
Here's the original page for this https://www.chromatic.com/build?appId=5d91884443f1dd00200b8a70&number=2